Documentation/ceph-dashboard-monitoring.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Ceph Dashboard Monitoring
+weight: 2500
+indent: true
+---
+{% include_relative branch.liquid %}
+
+## Ceph Dashboard Monitoring
+
+To enable monitoring and alerting functionality for the Ceph dashboard, additional steps are required to install the required dependencies.
+
+The monitoring stack components required include:
+
+1. [Prometheus operator](#prometheus-operator): records real-time metrics in a time series database
+2. [Prometheus Ceph manager module](#prometheus-ceph-manager-module): provide Ceph cluster metrics to Prometheus
+3. [Grafana](#grafana): displays charts for the information collected by Prometheus
+4. [Prometheus Node exporter](#prometheus-node-exporter): provides cluster nodes machine metrics to Prometheus
+5. [Prometheus alert manager](#prometheus-alert-manager): manage alerts produced in Prometheus
+
+## Production systems considerations
+
+The documentation provided is intended to explain how to provide the monitoring and alerting capabilities to the Ceph Dashboard. This is not replacing the original procedures/documentation that are needed to install the whole 
+monitoring stack in a kubernetes production system. Security, High availability, Multi Tenancy aspects are not in the scope of this document and those topics must be managed following the recommendations/documentation of each monitoring stack component.
+
+The monitoring stack installed in a Kubernetes production system is a separate functional component that provides a service not only to Ceph, but also to other applications. After the monitoring stack is configured, enabling monitoring and alerting in the Ceph Dashboard requires only two simple steps:
+
+1. Enable dashboard, prometheus manager module and monitoring in the [Ceph cluster CR](#ceph-cluster).
+2. Use the [Rook toolbox](ceph-toolbox.md) to set the urls used by the different monitoring stack components, and to disable (_set-*-api-ssl-verify settings_) the ssl certificates verification if self signed certificates are used. 
+```console
+ceph dashboard set-grafana-api-url <grafana_url>
```

Comment:
Where can the three urls be found to set in these commands? In the later sections of this doc? Seems like the specific commands should not be in this section since they need to read the later section to get the urls.